pkg/manifests/manifests.go

```diff
@@ -0,0 +1,193 @@
+package manifests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/http"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+func configureManifests(ctx *image.Context) error {
+	var downloadedManifests []string
+	var combinedManifestPaths []string
+	var extractedImages []string
+	var err error // created here to avoid scoping issues with "downloadedManifests" when using := on line 29
```

Comment:
There's no need to have the comment and comments listing a particular (or multiple) LoC are not generally a good idea since the implementation can easily change but the comment can remain untouched.

Reply:
👍